mono/mini/mini-runtime.c

```diff
@@ -900,6 +901,37 @@ mono_thread_abort (MonoObject *obj)
 	}
 }
 
+static void
+print_jit_counters ()
+{
+#if (defined(PLATFORM_MACOSX) || defined(__linux__) || defined(HOST_WIN32))
```

Comment:
No need for the conditional compilation here.
It's fine to print zero on unsupported targets.